examples/morpheusvm/cmd/morpheusvm/main.go

```diff
@@ -43,11 +52,54 @@ func main() {
 }
 
 func runFunc(*cobra.Command, []string) error {
-	if err := ulimit.Set(ulimit.DefaultFDLimit, logging.NoLog{}); err != nil {
+	logFactory := logging.NewFactory(logging.Config{
+		DisplayLevel: logging.Debug,
+	})
+
+	log, err := logFactory.Make("main")
+	if err != nil {
+		return fmt.Errorf("failed to initialize log: %w", err)
+	}
+
+	if err := ulimit.Set(ulimit.DefaultFDLimit, log); err != nil {
 		return fmt.Errorf("%w: failed to set fd limit correctly", err)
 	}
 
-	controller, err := controller.New()
+	txDBIndexer := indexer.NewTxDBIndexer(memdb.New())
```

Comment:
Using a memdb since it's an example vm + didn't want to add more code for cli flags to configure the db.